roles/openshift_master/tasks/main.yml

```diff
@@ -87,6 +87,9 @@
       master_count: "{{ openshift_master_count | default(None) }}"
       controller_lease_ttl: "{{ osm_controller_lease_ttl | default(None) }}"
       master_image: "{{ osm_image | default(None) }}"
+    role: common
+    local_facts:
+      imageconfig_latest: "{{ openshift_imageconfig_latest | default(False) }}"
```

Comment:
The imageconfig_latest fact should be added in the openshift_common role instead of the openshift_master role...  Since openshift_common is a dependency of openshift_master, it will be picked up just the same as if it was added in this role.